Makefile

```diff
@@ -101,27 +101,30 @@ build-fat build-full build: $(CODER_FAT_BINARIES)
 release: $(CODER_FAT_BINARIES) $(CODER_ALL_ARCHIVES) $(CODER_ALL_PACKAGES) $(CODER_ARCH_IMAGES) build/coder_helm_$(VERSION).tgz
 .PHONY: release
 
-build/coder-slim_$(VERSION)_checksums.sha1 site/out/bin/coder.sha1: $(CODER_SLIM_BINARIES)
+build/coder-slim_$(VERSION)_checksums.sha1: site/out/bin/coder.sha1
+	cp "site/out/bin/coder.sha1" "build/coder-slim_$(VERSION)_checksums.sha1"
```

Comment:
should use the make shortcuts for "first dependency" `$<`and "target" `$@` here

Reply:
I'm afraid I don't know how to use it. Would it look like this?

```make
build/coder-slim_$(VERSION)_checksums.sha1: site/out/bin/coder.sha1
	cp $< $@
```

Reply:
with quotes!!!!!

Reply:
looks like you quoted in your commit 👍

Reply:
See the commit 😉